src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <div className="App">
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+      {person.age ? <p className="Person__age">I am {person.age}</p> : ''}
+      {person.isMarried ? (
+        <p className="Person__partner">
```

Comment:
`<p className="Person__partner">` in both cases, only the content is different

Reply:
I can not delete "Person__partner", because it is broke test